lesson16/db.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
MUST
The json structure is not realistic.
When I first tried this, I thought of categories and articles separately, but in fact this is how it works.
It is an image of information all contained within a single object.
The following is based on a sample shown by moriken-sensei on Discord.
```js
  {
    "category": "News",
    "id": // 適当な文字列
    "display": true, //どのカテゴリタブを初期表示時に選んでいるかはデータとして持っている
    "img": "./images/bg-news.png",
    "article": [
      {
        "date": "2022/02/24",
        "title": "News01",
        "id": // 適当な文字列
        "comment": [
          {
            "name": "Harry",
            "id": // 適当な文字列
            "date": "2022/02/24"
          },
          {
            "name": "James",
            "id": // 適当な文字列
            "date": "2022/02/25"
          }
        ]
      },
```

Reply:
Thank you for the review.

Thank you very much for your kind words about the json structure.
This is fixed in the following commit.

[e917b64](https://github.com/nyattsu72/hanson-frontend-cource/pull/21/commits/e917b642cce0f779ed268638993f93e220f5234f)